tools/cosmovisor/cmd/cosmovisor/batch_upgrade.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+func NewBatchAddUpgradeCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:          "add-batch-upgrade <upgrade1-name>:<path-to-exec1>:<upgrade1-height> .. <upgradeN-name>:<path-to-execN>:<upgradeN-height>",
```

Comment:
Let's improve the UX and let it take a batch file instead (we can always fall back to the multi args if the file doesn't exist)

Reply:
How about the following format:
```
cosmovisor add-batch-upgrade --upgrade-file <path-to-batch-file>
```
As well as:
```
cosmovisor add-batch-upgrade --upgrade-list <upgrade1-name>:<path-to-exec1>:<upgrade1-height> .. <upgradeN-name>:<path-to-execN>:<upgradeN-height>
```
Users can use one of these two methods to provide batch upgrades.